packages/dos-donts-block/src/DoDontItem.tsx

```diff
@@ -344,25 +330,31 @@ export const SortableDoDontItem = (props: SortableDoDontItemProps) => {
     const { attributes, listeners, setNodeRef, setActivatorNodeRef, transform, transition, isDragging } = useSortable({
         id,
     });
-
+    const [draggableProps, setDraggableProps] = useState<Record<string, unknown>>(
+        editing ? { ...attributes, ...listeners } : {}
+    );
     const transformStyle = {
         transform: transform ? `translate(${transform.x}px, ${transform.y}px)` : '',
         transition,
         zIndex: isDragging ? 2 : 1,
     };
 
-    const draggableProps = editing ? { ...attributes, ...listeners } : {};
+    useEffect(() => {
+        if (!isDragging) {
+            setDraggableProps(editing ? { ...attributes, ...listeners } : {});
+        }
+    }, [isDragging, attributes, listeners, editing]);
 
     return (
-        <DoDontItem
-            key={id}
-            ref={setNodeRef}
-            {...props}
-            isDragging={isDragging}
-            replaceWithPlaceholder={isDragging}
-            transformStyle={transformStyle}
-            draggableProps={draggableProps}
-            setActivatorNodeRef={setActivatorNodeRef}
-        />
+        <div style={transformStyle} ref={setNodeRef}>
```

Comment:
Previously the transformStyle was being overwritten with zIndex:0, not sure if this is now not required here